docs/developing_recipes_locally.md

```diff
@@ -0,0 +1,207 @@
+# Developing Recipes Locally
+
+One of the early steps for contributing get getting familiar with the Conan client. There's also a few steps you need to take to get your environment setup.
```

Comment:
I'm not quite sure what is meant by the first sentence, `One of the early steps for contributing get getting familiar with the Conan client.`, here. Maybe a typo?

Reply:
```suggestion
Before you can contribute any code changes, you'll need to make sure you are familiar with the Conan client and have an environment that is conducive to developing recipes.
```